corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui_advanced.js

```diff
@@ -576,6 +576,9 @@ hqDefine('app_manager/js/forms/case_config_ui_advanced', function () {
                 if (action.auto_select) {
                     nameSnip = "<i class=\"fa fa-tag\"></i> <%= action.case_tag() %> (autoselect mode: <%= action.auto_select.mode() %>)";
                 }
+                if (action.load_case_from_fixture) {
+                    nameSnip += " (from fixture)";
```

Comment:
does this need to be translated?

Reply:
Yeah, good point. Did a pass on the file to add translations.